dist/templates/ovnkube-master.yaml.j2

```diff
@@ -137,6 +137,46 @@ spec:
         lifecycle:
       # end of container
 
+      - name: run-nbctld
```

Comment:
@girishmg does it really need to be another container, or can we just run it in the OVN container if one exists or on the host itself (for external OVN mode)?

Reply:
nevermind, using the container restart semantics is a nice way of handling its lifecycle